language/tools/move-package/src/resolution/dependency_graph.rs

```diff
@@ -415,6 +438,23 @@ impl DependencyGraph {
             bail!("Merging dependencies into a graph after calculating its 'always' dependencies");
         }
 
+        // Because all the packages in `extensions`'s package table didn't exist in `self`'s, all
```

Comment:
After your change to handle overlapping graphs, is this still the case?  I think the doc-comment on `merge` is now out-of-date.

Reply:
I am not sure which part of `merge`'s doc comment you have in mind. The one on line 441 above is still correct I think as we just moved adding edges from extension graph to the end of graph processing so the invariant about edges not existing in the original graph still holds.